sdk/test-common/src/main/daml/upgrades/FailsWhenDepsDowngradeVersions/v1/Main.daml

```diff
@@ -0,0 +1,9 @@
+-- Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Main where
+
+import Dep
+
+self : Text
+self = "v1 with " <> dep
```

Comment:
We have to actually use the dependency, otherwise it won't show up during participant validation in `directDeps`